src/registry/local.rs

```diff
@@ -137,10 +140,14 @@ mod tests {
             package_bytes
         );
 
+        let registry_uri = RegistryUri::from_str("http://some-registry/artifactory")
+            .expect("Failed to parse registry URL");
```

Comment:
Im the registry uri type is fine – if it does validate the `*.jfrog.io` rule + protocol assumptions while parsing!

Reply:
Yup, I'll give it a constructor instead.